src/docs/product/cli/dif.mdx

```diff
@@ -273,12 +273,12 @@ sentry-cli upload-proguard \
 ```
 
 Since the Android SDK needs to know the UUID of the mapping file, you need
-to embed it in a `sentry-debug-meta.properties` file. If you supply
-`--write-properties` that is done automatically:
+to associate it with the upload. If you supply
+`--uuid` sets that value:
```

Comment:
Seems like you're missing a word here. I think this corrects this:

```suggestion
`--uuid`, it sets that value:
```

Reply:
Thanks, you're right. Added.